include/libopencm3/stm32/h7/flash.h

```diff
@@ -0,0 +1,56 @@
+/** @defgroup flash_defines FLASH Defines
+ *
+ * @ingroup STM32H7xx_defines
+ *
+ * @author @htmlonly &copy; @endhtmlonly 2019
+ * Brian Viele <vielster@allocor.tech>
+ *
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_FLASH_H
+#define LIBOPENCM3_FLASH_H
+
+#include <libopencm3/stm32/common/flash_common_all.h>
+#include <libopencm3/stm32/common/flash_common_f.h>
+#include <libopencm3/stm32/common/flash_common_f24.h>
+
+/**@{*/
+
+/* --- FLASH_ACR values ---------------------------------------------------- */
```

Comment:
you don't need lines like these when you h ave the doxygen groupings, you can just drop them.